src/scheduler/fifo.cpp

```diff
@@ -1013,7 +1013,7 @@ main_sched_loop(status *policy, sched_svrconn *sconn, server_info *sinfo, schd_e
 				 */
 				clear_schd_error(chk_lim_err);
 				if (sinfo->qrun_job == NULL) {
-					chk_lim_err->error_code = (enum sched_error)check_limits(sinfo,
+					chk_lim_err->error_code = (enum sched_error_code)check_limits(sinfo,
```

Comment:
In other places you make a static_cast to enum sched_error_code.  Here you use a C styler cast.  Shouldn't we be consistent?

Reply:
you're right, i'll fix the others as well.

Reply:
nvm, it's just the one